proto/florin/v1/query.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package florin.v1;
+
+import "amino/amino.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/monerium/module-noble/v2/types/v1";
+
+service Query {
+  rpc MintAllowance(QueryMintAllowance) returns (QueryMintAllowanceResponse) {
+    option (google.api.http).get = "/florin/v1/mint_allowance/{denom}/{account}";
+  }
+}
+
+//
+
+message QueryMintAllowance {
+  string denom = 1;
+  string account = 2;
+}
+
+message QueryMintAllowanceResponse {
+  string allowance = 1 [
+    (amino.dont_omitempty) = true,
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
```suggestion
  string allowance = 1;
```

@gislik Try this to fix your issue! Don't forget to run `make proto-all` 🙃

Reply:
Will do.